recipes/cabinetry/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "cabinetry" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 88accfd0a6138e1b0c7ce0b5f44a746a53ebed8488bea02a5333846fee71f30a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - cabinetry = cabinetry.cli:cabinetry
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - pyhf[minuit] ~=0.6.3
+    - boost_histogram >=1.0.0
+    - awkward >=1.0
+    - tabulate >=0.8.1
+    - matplotlib-base
+    - typing_extensions
+
+test:
+  imports:
+    - cabinetry
+  commands:
+    - cabinetry --help
+
+about:
+  home: https://github.com/scikit-hep/cabinetry
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'library for building and steering binned template fits'
+
+  description: |
+    cabinetry is a Python library for building and steering binned template fits. 
+    It is written with applications in High Energy Physics in mind. 
+    cabinetry interfaces many other powerful libraries to make it easy for an analyzer to run their statistical inference pipeline.
+    Statistical models in HistFactory format can be built by cabinetry from instructions in a declarative configuration. 
+    cabinetry makes heavy use of pyhf for statistical inference, and provides additional utilities to help study and disseminate fit results. 
+    This includes commonly used visualizations. Due to its modular approach, analyzers are free to use all of cabinetry's functionality or only some pieces.
+    cabinetry can be used for inference and visualization with any pyhf-compatible model, whether it was built with cabinetry or not.
+  doc_url: https://cabinetry.readthedocs.io/en/latest/
+  dev_url: https://github.com/scikit-hep/cabinetry
+
+extra:
+  recipe-maintainers:
+    - alexander-held
+    - matthewfeickert
```

Comment:
I confirm that I am willing to be a maintainer of `cabinetry` on conda-forge. :+1: